projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -23,14 +23,16 @@
             class="input-phone tui-group__auto-width-item"
             tuiTextfieldAutocomplete="off"
             [tuiTextfieldLabelOutside]="true"
-            [formControl]="control"
+            [ngModel]="value"
             [focusable]="focusable"
             [countryCode]="inputPhoneCountryCode"
             [phoneMaskAfterCountryCode]="phoneMaskAfterCountryCode"
             [readOnly]="readOnly"
-            [pseudoInvalid]="pseudoInvalid"
+            [pseudoInvalid]="computedInvalid"
             [pseudoFocused]="pseudoFocused"
             [pseudoHovered]="pseudoHovered"
+            (ngModelChange)="onModelChange($event)"
+            (focusedChange)="onFocusChange($event)"
```

Comment:
This is probably wrong. We need to rely on `ActiveZone` for this, because focusing countries dropdown is also considered focusing this component

Reply:
:heavy_check_mark: